articles/_posts/2015-08-03-on-a-moratorium-on-new-browser-features.md

```diff
@@ -106,8 +106,9 @@ _This was written by Bruce Lawson, with input from the rest of Opera’s Develop
 
 Other people have been commenting, too:
 
-* Simultaneously with our publishing, Google Chrome evangelist Jake Archibald wrote [If we stand still, we go backwards](https://jakearchibald.com/2015/if-we-stand-still-we-go-backwards/)
+*  Simultaneously with our publishing, Google Chrome evangelist Jake Archibald wrote [If we stand still, we go backwards](https://jakearchibald.com/2015/if-we-stand-still-we-go-backwards/)
```

Comment:
no idea what you mean. check again ...

On 7 August 2015 at 18:49, Vadim Makeev <notifications@github.com> wrote:

> In articles/_posts/2015-08-03-on-a-moratorium-on-new-browser-features.md
> <https://github.com/operasoftware/devopera/pull/326#discussion_r36543016>:
>
> > @@ -106,8 +106,9 @@ _This was written by Bruce Lawson, with input from the rest of Opera’s Develop
> >
> >  Other people have been commenting, too:
> >
> > -* Simultaneously with our publishing, Google Chrome evangelist Jake Archibald wrote [If we stand still, we go backwards](https://jakearchibald.com/2015/if-we-stand-still-we-go-backwards/)
> > +*  Simultaneously with our publishing, Google Chrome evangelist Jake Archibald wrote [If we stand still, we go backwards](https://jakearchibald.com/2015/if-we-stand-still-we-go-backwards/)
>
> Double space is back!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/operasoftware/devopera/pull/326/files#r36543016>.
>